apps/api/src/schema/websoc.ts

```diff
@@ -75,18 +75,36 @@ const isValidRestrictionCode = (code: string): code is (typeof restrictionCodes)
 export const websocQuerySchema = z.object({
   year: z
     .string({ message: "Parameter 'year' is required " })
-    .length(4, { message: "Parameter 'year' must have length 4" }),
-  quarter: z.enum(terms, { required_error: "Parameter 'quarter' is required" }),
+    .length(4, { message: "Parameter 'year' must have length 4" })
+    .openapi({ description: "The academic year of the course ", example: "2025" }),
+  quarter: z.enum(terms, { required_error: "Parameter 'quarter' is required" }).openapi({
+    description: "The academic quarter of the course",
```

Comment:
Again, this is worded like it's a property of a single course returned by the endpoint when it really exists in the query to filter an unknown number of courses. Think of this as one of the most broad parameters of the query. Something like: "The quarter in which to search".